fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -0,0 +1,250 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.scheduler;
+
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.analysis.SetVar;
+import com.starrocks.analysis.StringLiteral;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.qe.QueryState;
+import com.starrocks.qe.SessionVariable;
+import com.starrocks.qe.VariableMgr;
+import com.starrocks.statistic.Constants;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.Future;
+
+public class TaskRun implements Writable {
```

Comment:
I think TaskRun don't need to be serializable, only TaskRunStatus needs to be serialized.